certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -630,11 +630,32 @@
             "x-ms-visibility": "important",
             "type": "string"
           },
+          {
+            "name": "verificationType",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Verification Type",
+            "x-ms-summary": "Verification Type",
+            "x-ms-dynamic-values": {
+              "operationId": "StaticResponseForVerificationTypes",
+              "value-collection": "verificationTypes",
+              "value-path": "type",
+              "value-title": "name"
+            }
+          },
+          {
+            "name":"routingOrder",
```

Comment:
@tle-docusign we still have this issue. You promoted routingorder to always show up in the UI. It shoudl be under advanced options. Just remove it from here, and restore it to under "AdditionalRecipientData".